unittest-statistics.py

```diff
@@ -20,26 +20,25 @@ def main():
         if not line:
             break
         line = line.decode()
-        m = re.match('.*"(.*)".*: ([0-9]+)(us|mks|ms)', line)
+        m = re.match('(.*):.*: Leaving test case.*"(.*)".*: ([0-9]+)(us|mks|ms)', line)
```

Comment:
Why are there two colons?

The output with Boost version 1.66.0 is

```
test/checkqueue_tests.cpp(275): Leaving test case "test_CheckQueue_UniqueCheck"; testing time: 159458us

Reply:
Interesting. On my setup I get output like:

`test/bswap_tests.cpp:12: Leaving test case "bswap_tests"; testing time: 1924us
test/bswap_tests.cpp:10: Leaving test suite "bswap_tests"; testing time: 1968us`

I'd say it's safer to make less assumptions about the output format. Since the source file name is not really necessary. Maybe I should stick to the minimum sufficient change to fix the problem on my setup, which would be to add the string "Leaving test case" to the regular expression, and remove other changes. BTW this "Leaving test case" message is documented in the boost docs, so I think it's a safe change.

I'll amend the commit accordingly.

Reply:
Agree. Thx

Reply:
No problem. Amended the commit. thx.